specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json

```diff
@@ -1228,6 +1228,283 @@
+        "tags": [
+          "LinkedWorkspaces"
+        ],
+        "operationId": "LinkedWorkspaces_Create",
+        "description": "Add a new linked workspace.",
+        "x-ms-examples": {
+          "CreateLinkedWorkspace": {
+            "$ref": "./examples/createLinkedWorkspace.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The object for adding a new linked workspace",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/LinkedWorkspaceCreateDto"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully created the linked workspace.",
+            "schema": {
+              "$ref": "#/definitions/LinkedWorkspace"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/Microsoft.MachineLearningServices/workspaces/{workspaceName}/linkedWorkspaces/{linkName}": {
+      "get": {
+        "tags": [
+          "LinkedWorkspaces"
+        ],
+        "operationId": "LinkedWorkspaces_Get",
+        "description": "Get the detail of a linked workspace.",
+        "x-ms-examples": {
+          "GetLinkedWorkspace": {
+            "$ref": "./examples/getLinkedWorkspace.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LinkName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the detail of the linked workspace.",
+            "schema": {
+              "$ref": "#/definitions/LinkedWorkspace"
+            }
+          },
+          "404": {
+            "description": "Specific linked workspace not found.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "post": {
```

Comment:
>post [](start = 7, length = 4)

Resource updates should be done with PUT not POST as per the RPC. ([https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md))

Reply:
again, that's because LinkedWorkspace is not a resource, yet.